packages/fxa-content-server/app/scripts/models/user.js

```diff
@@ -568,99 +568,29 @@ var User = Backbone.Model.extend({
   },
 
   /**
+   * Disconnect an attached client from the given account.
    *
-   * @param {Object} account - account object
-   * @param {Object} client - an attached client
-   * @returns {Promise}
-   */
-  destroyAccountClient (account, client) {
-    if (client.get('clientType') === Constants.CLIENT_TYPE_DEVICE) {
-      return this.destroyAccountDevice(account, client);
-    } else if (client.get('clientType') === Constants.CLIENT_TYPE_OAUTH_APP) {
-      return this.destroyAccountApp(account, client);
-    } else if (client.get('clientType') === Constants.CLIENT_TYPE_WEB_SESSION) {
-      return this.destroyAccountSession(account, client);
-    }
-  },
-
-  /**
-   * Fetch the devices for the given account, populated the passed in
-   * Devices collection.
-   *
-   * @param {Object} account - account for which device list is requested
-   * @param {Object} devices - Devices collection used to store list.
-   * @returns {Promise} resolves when the action completes
-   */
-  fetchAccountDevices (account, devices) {
-    return account.fetchDevices(devices);
```

Comment:
By the way, these were all being called in a confusing manner, `account.fetchDevices` returns the list rather than populating it into the passed argument.